### PR TITLE
[WIP] Updated feature list

### PR DESCRIPTION
**Pull request by MGBla**

### CODE REVIEW

File: ur_robot_driver/doc/features.md [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
+| scaled position-based control                         | yes                       |
+| velocity-based control                                | no<sup>1</sup>            |
+| reporting of tcp wrench                               | not yet implemented       |
+| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| pausing of programs                                   | yes                       |
+| continue trajectories after EM-Stop resume            | yes                       |
+| continue trajectories after protective stop           | yes                       |
+| panel interaction in between possible                 | yes                       |
+| get and set IO states                                 | yes                       |
+| use tool communication on e-series                    | not yet tested            |
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:15]: Maybe add a link to https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap

File: ur_robot_driver/doc/features.md [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
 
-<sup>1</sup> Depending on the mode the driver is running the panel won't react or using the panel
-will stop the program without notifying the ROS user.
-
-<sup>2</sup> In velocity mode this is implicitly given.
-
-<sup>3</sup> There is no specific plan to do this inside of the first driver development. However,
-it is structured in a way so that a ROS2 driver should be developed as easy as possible by keeping
-as much as possible in a ros-independent library.
+<sup>1</sup> Currently not possible in ROS2 control. The driver will be extended as soon as the velocity controller is available
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:25]: Will suggest you add a version to the ROS2 control and remove the promise. Thus it is the intention.

**Review by urrsk — CHANGES_REQUESTED** [review]
Looks good. Just these two comments

File: ur_robot_driver/doc/features.md [hunk 3]
```diff
@@ -1,32 +1,26 @@
...
+| velocity-based control                                | no<sup>1</sup>            |
+| reporting of tcp wrench                               | not yet implemented       |
+| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| pausing of programs                                   | yes                       |
+| continue trajectories after EM-Stop resume            | yes                       |
+| continue trajectories after protective stop           | yes                       |
+| panel interaction in between possible                 | yes                       |
+| get and set IO states                                 | yes                       |
+| use [tool communication forwarder](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCap) on e-series | should work, not yet tested with ROS2            |
+| use the driver without a teach pendant necessary      | yes                       |
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:16]: Has this been verified?

**Review by urrsk — APPROVED** [review]
I added the link to the ToolComm_Forwarder.
That is blocking us for saying that it is supported?
```socat``` is independing of ROS2 and simulates a serial port.
And serial port is support by ROS2, right?

File: ur_robot_driver/doc/features.md (hunk 3 above)
Comment (MGBla) [reply ur_robot_driver/doc/features.md:16]: Not yet, I don't have any necessary hardware at home. Will try to organize some some in the near future. Also we should provide a ROS2 Version of the [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/tool_communication) for easier use.

**Pull request merged**